tests/PHPUnit/Framework/Fixture.php

```diff
@@ -388,6 +387,8 @@ public static function installAndActivatePlugins()
                 $pluginsManager->activatePlugin($name);
             }
         }
+
+        $pluginsManager->loadPluginTranslations();
```

Comment:
I thought this happened automatically, but I can check again.

The problem I had w/ translations is that some tests require translations (like many system tests), and some tests don't test against actual translations. Disabling it or enabling it for everyone breaks some tests. I think I will go towards testing against tokens by default.